src/modules/add-and-remove.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable import/prefer-default-export */
```

Comment:
- [ ] Please, consider removing the linter disabled. Let the linter makes its direct job - find the bugs in the early steps of your project.